Doc/Tutorial/chapter_entrez.tex

```diff
@@ -391,10 +390,10 @@ \section{EFetch: Downloading full records from Entrez}
 from Bio import SeqIO
 from Bio import Entrez
 Entrez.email = "A.N.Other@example.com"  # Always tell NCBI who you are
-filename = "gi_186972394.gbk"
+filename = "Acc_EU490707.gbk"
```

Comment:
Just ``EU490707.gbk`` as the filename would be fine and consistent with NCBI usage.